gtsam/inference/Ordering.cpp

```diff
@@ -287,6 +287,17 @@ void Ordering::print(const std::string& str,
   cout.flush();
 }
 
+/* ************************************************************************* */
+Ordering::This& Ordering::operator+=(KeyVector& keys) {
```

Comment:
@dellaert FYI since this is a GTSAM change.

Reply:
Not a change, but just some added functionality. Nothing will break but we can now do the python equivalent of a list extend.